packages/env/lib/wordpress.js

```diff
@@ -44,19 +44,9 @@ async function checkDatabaseConnection( { dockerComposeConfigPath, debug } ) {
  * @param {Object}        spinner     A CLI spinner which indicates progress.
  */
 async function configureWordPress( environment, config, spinner ) {
-	const url = ( () => {
-		const port = config.env[ environment ].port;
-		const domain =
-			environment === 'tests'
-				? config.env.tests.config.WP_TESTS_DOMAIN
-				: config.env.development.config.WP_SITEURL;
-		if ( port === 80 ) {
-			return domain;
-		}
-
-		return `${ domain }:${ port }`;
-	} )();
-
+	const url =
+		config.env[ environment ].config?.WP_SITEURL ??
+		`http://localhost:` + config.env[ environment ].port;
```

Comment:
We could just save the URL to the config object to avoid generating it in multiple places